src/content/developers/docs/gas/index.md

```diff
@@ -36,11 +36,50 @@ Although a transaction includes a limit, any gas not used in a transaction is re
+
+High gas fees are due to the popularity of Ethereum. To perform any operation on Ethereum requires consuming gas. This includes calculations, storing or manipulating data, or transferring tokens, each consuming different amounts of "gas" units. As dapp functionality grows more complex, the number of operations a smart contract performs grows too, and all contribute to high amounts of gas that must be paid for.
+
+Gas is paid for with small amounts of ether (denominated in gwei, or (10<sup>-9</sup> ETH)). This value represents the amount of ether being paid per "unit of gas", not for the entire transaction. Since gas is capped within a block, under the current system, this means that users must out-bid one another to be included in the next block if demand exceeds capacity. Gas cost alone does not actually determine how much we have to pay for a particular transaction. To calculate the transaction fee we have to multiply the gas cost by gas price. (Which is measured in gwei)
+
+This video about gas fees explains fully why fees are so expensive:
+
+<iframe width="100%" height="315" src="https://www.youtube.com/watch?v=Yh8cHUB-KoU" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Initiatives to reduce gas costs {#Initiatives-to-reduce-gas-costs}
+
+With the new network upgrades of Ethereum 2.0 (also known as Eth2 or Serenity). This should ultimately address some of the gas fee issues, which will in turn enable the platform to process thousands of transactions per second and scale globally.
+
+Layer 2 scaling is a primary initiative to greatly improve gas, user experience and scalability, our page on layer 2 scaling and rollups can be found here:
+-[Layer 2 scaling](https://ethereum.org/en/developers/docs/layer-2-scaling/)
+
+The new proof-of-stake model should reduce high power consumption and reliance on specialized hardware. The new PoS system was introduced on the Beacon Chain. This chain will allow the decentralized Ethereum network to come to agreement and keep the network secure, but avoid high energy use by requiring a financial commitment. 
+
+Anyone with at least 32 ETH is able to stake them and become a validator responsible for processing transactions, adding new block to the block chain and storing data. Users who have less than 32 ETH will be able to join staking pools.
+
+## What is Gas limit? {What-is-gas-limit}
+Gas limit refers to the maximum amount of gas you are willing to spend on a transaction. A higher gas limit means more computational work must be done to excecute the [smart contract](https://ethereum.org/en/developers/docs/smart-contracts/). A standard ETH transfer requires a gas limit of 21,000 units of gas.
+
+If you specify too little gas this can be risky as if the gas runs out before your transaction is complete, since you have lost that gas everything reverts.
```

Comment:
"if the gas runs out before your transaction is complete, since you have lost that gas everything reverts." 
confusing wording... see above, if you can find a way to simplify this.

Reply:
I have merged "if the gas runs out before your transaction is complete, since you have lost that gas everything reverts." with the info you gave above. So it now looks like this 

"For example if you put a gas limit of 50,000 for a simple ETH transfer, the EVM would consume 21,000, and you would get back the remaining 29,000. However, if you specift too little gas sat for example, a gas limit of 20,000 for a simple ETH transfer, the EVM will consume your 20,000 gas units to fulfill the txn, but it will not complete. The EVM then reverts any changes, but since 20k gas units worth of work has already been done by the miner, that gas is consumed."